docs/reference/camera.md

```diff
@@ -293,6 +293,8 @@ period = wb_camera_get_sampling_period(tag)
 *enable and disable camera updates*
 
 The `wb_camera_enable` function allows the user to enable a camera.
+Once the camera is enabled, it will copy images from GPU memory to CPU memory at each time step, regardless of `wb_camera_get_image` calls.
```

Comment:
> But when testing a little bit I found a crash:
> 
> 1. open `range_finder.wbt`
> 2. change the RangeFinder image width and set it to a bigger value (for example 257)
> 3. change the RangeFinder.spherical value to TRUE
> 
> -> Webots crashes

Thank you for discovering this, I can reproduce it by running the simulation and then changing the width and spherical parameters. However, the same happens on the develop branch, so it is not regression.

Should I fix it in this PR or another?

Reply:
> Should I fix it in this PR or another?

If it doesn't create any conflicts with this branch, it should be fixed in another PR, otherwise it is probably better to fix it in this PR.